l3_dump.py

```diff
@@ -46,18 +46,19 @@
     READELF_DATA_SECTION = '__cstring'
 
 # #############################################################################
-def set_decode_loc_id(program_bin:str):
+def set_decode_loc_id(program_bin:str, loc_decoder_bin:str):
```

Comment:
Shouldn't `loc_decoder_bin`'s type be `str | None`?

Reply:
Could be ... I changed it to:

```
 49 def set_decode_loc_id(program_bin:str, loc_decoder_bin:str = None):
```

... which is probably a bit more self-documenting.